micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/SidebarAdd.js

```
@@ -0,0 +1,571 @@
+            .then(text => {
+                const icons = extractExports(text);
+                const iconObjects = icons.map(icon => ({ code: icon }));
+                setIconNames(iconObjects);
+            });
+
+    }, [])
+
+    const { data: USER_ROLE } = Digit.Hooks.useCustomMDMS(tenantId, "ACCESSCONTROL-ROLES", [{ name: "roles" }], {
+        select: (data) => {
+            return data;
+            ;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)